lib/mix/tasks/firmware.burn.ex

```diff
@@ -51,7 +52,7 @@ defmodule Mix.Tasks.Firmware.Burn do
       Environment variable $NERVES_TOOLCHAIN is not set
     """
 
-    fw = Path.join(File.cwd!, "_images/#{target}/#{otp_app}.fw")
+    fw = Path.join(File.cwd!, "#{images_path}/#{target}/#{otp_app}.fw")
```

Comment:
also, in this case we would not need target anymore since the default build path would contain it.